frontend/src/app/main/data/workspace/changes.cljs

```diff
@@ -34,6 +34,25 @@
 
 (declare commit-changes)
 
+
+(defn- add-changes-group-id
+  [changes state]
+  (let [undo                  (:workspace-undo state)
+        items                 (:items undo)
+        index                 (or (:index undo) (dec (count items)))
+        prev-item             (when-not (or (empty? items) (= index -1))
+                                (get items index))
+        changes-group-id      (:changes-group-id prev-item)
+        add-changes-group-id? (and
+                               (not (nil? changes-group-id))
+                               (= (get-in changes [:redo-changes 0 :type]) :mod-obj)
+                               (= (get-in prev-item [:redo-changes 0 :type]) :add-obj)) ;; This is a copy-and-move with mouse+alt
+        ]
+    (if add-changes-group-id?
+      (assoc changes :changes-group-id changes-group-id)
```

Comment:
the `:changes-group-id looks redundant` on datastructure called `changes`, maybe `:group-id` ?

Reply:
Ok, changed